src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -338,6 +340,23 @@ private List<int> FindPeersToResync(ChainedHeader consensusTip)
             return peerIdsToResync;
         }
 
+        /// <summary>Find the fork between two headers and return the fork if the headers are not on the same chain.</summary>
+        private ChainedHeader FindForkNotOnConensusChain(ChainedHeader peerTip, ChainedHeader consensusTip)
```

Comment:
You have mismatch of the summary and method name. It should be named FindForkIfChainedHeadersNotOnSameChain and your parameters should be named header1 and header2 as to be agnostic of what they represent.

Reply:
ok sounds fine I will fix